presto-main/src/main/java/io/prestosql/sql/planner/planprinter/PlanRepresentation.java

```diff
@@ -66,6 +66,18 @@ public TypeProvider getTypes()
         return Optional.ofNullable(nodeInfo.get(id));
     }
 
+    /**
+     * Adds a node in query plan tree.
+     * <p>
+     * A query plan tree represents the data that must be retrieved by a query,
+     * with each node representing an operation. As an example of sql plan tree:
+     * - Aggregate(FINAL)[regionkey] => [regionkey:bigint, count:bigint]
+     * -- LocalExchange[HASH][$hashvalue] ("regionkey") => regionkey:bigint, count_8:bigint, $hashvalue:bigint
+     * -----RemoteExchange[REPARTITION][$hashvalue_9] => regionkey:bigint, count_8:bigint, $hashvalue_9:bigint
+     * --------Project[] => [regionkey:bigint, count_8:bigint, $hashvalue_10:bigint]
```

Comment:
Makes sense to add pipes like this so it looks more like a tree?
```suggestion
     *        |-----Project[] => [regionkey:bigint, count_8:bigint, $hashvalue_10:bigint]
```